cmd/frontend/build_frontend_env/build-frontend-env.go

```diff
@@ -0,0 +1,81 @@
+package build_frontend_env
+
+/*
+$ your_cli_tool build-frontend-env --secret "YourSecretName" --bucket "YourBucketName"
```

Comment:
why is `YourBucketName` a parameter here

Reply:
The bucketName parameter is included in the code as a placeholder for the name of an S3 bucket.  It was added as a potential placeholder for situations where we might need to include the name of an S3 bucket in our .env file.

LMK if i truly need this or not though. had a similar question for daniel. @karkir0003

Reply:
If you need the name of an S3 bucket, you can probably add a constant within DLP repo or in parameter store?

Reply:
got it.

Reply:
@karkir0003 Should I use` dlp-upload-bucket` as name of S3 bucket, as you mentioned before that it  is the name of S3 bucket responsible for storing user uploaded dataset files? I can probably add this as a constant.

Reply:
that's fine. you can add `dlp-upload-bucket` as a constant

Reply:
ok awesome, it works now.